src/content/docs/docs/intro/web-monetization-extension.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: Web Monetization extension
+---
+
+import { Steps } from '@astrojs/starlight/components';
+import { LinkOut } from "@interledger/docs-design-system";
+
+The Interledger Foundation's Web Monetization extension allows you to support your favorite web monetized sites directly from your web browser in a quick, non-intrusive, and privacy-preserving way.
+
+### Web monetized sites
+
+To use the extension to make Web Monetization payments, the site you visit must be web monetized. Since Web Monetization is fairly new, we encourage content owners and publishers tell their audience they're web monetized via a banner or some other method.
+
+If you're interested in web monetizing your own content, the [Add Web Monetization to a page](/docs/guides/monetize-page) guide is a good place to start.
+
+### Web Monetization payments 
+
+To make Web Monetization payments, you only need an account with a [Web Monetization-enabled wallet provider](/docs/resources/op-wallets). There's no site-by-site account or subscription required for each individual site you want to support.
+
+A web monetized site cannot initiate payments from your digital wallet. All payments are explicitly initialized and authorized by you. The sites you choose to pay are not provided with any of your personally identifiable information, such as your name, email address, or IP address. 
+
+:::note
+Sending a payment to a web monetized site does not guarantee an ad-free experience. The decision to show or hide ads in response to a Web Monetization payment is entirely up to each content owner or publisher.
+:::
+
+## Set up the extension
+
+### 1. Download and install the extension
+
+The extension is currently supported in the following browsers. We plan to support more browsers in the future. 
+
+* <LinkOut href="https://chromewebstore.google.com/detail/web-monetization-extensio/oiabcfomehhigdepbbclppomkhlknpii">Google Chrome and Chromium-based browsers</LinkOut>
```

Comment:
Not related to this PR, but the text of this link doesn't wrap, causing a scroll on mobile. Due to this CSS somewhere:
```css
.icon {
  white-space: nowrap;
  display: inline-flex;
}
```

Reply:
I think it's caused by our homegrown LinkOut component. I was able to see the same behavior on the Rafiki site on my phone. I'll create an issue for Sarah.